ui/src/components/SampleControls/SampleControls.module.css

```diff
@@ -101,16 +87,36 @@
   font-size: 25px;
 }
 
-.overlay {
-  position: absolute;
-  z-index: 1001;
-  display: flex;
-  margin-top: 0.5rem;
-  padding: 0.625rem 0.75rem;
-  background-color: rgba(0, 0, 0, 0.6);
-  border-radius: 4px;
```

Comment:
This all goes away except the custom background colour thanks to `Popover` (cf. `.popover`). The background colour is now defined via Bootstrap's `--bs-popover-bg` and matches the background colour of the sample controls bar (slightly grey instead of black so that when the sample video is black, the overlays still stand out).